clients/include/solver/testing_getrf_batched.hpp

```diff
@@ -48,9 +48,10 @@ void testing_getrf_batched_bad_arg(const Arguments& arg)
     int64_t            lda         = 102;
     int64_t            batch_count = 2;
     int64_t            A_size      = N * lda;
-    int64_t            Ipiv_size   = std::min(M, N) * batch_count;
+    int64_t            Ipiv_size   = std::min(M, N);
 
-    device_batch_vector<T> dA(A_size, 1, batch_count);
+    // Allocate device memory
+    device_batch_matrix<T> dA(M, N, lda, batch_count);
     device_vector<int>     dIpiv(Ipiv_size);
```

Comment:
Isn't the ipiv_size supposed to be * batch_count?     Not sure why they differ in style from bad_arg to normal test either.

Reply:
Yes it is, will change it. I think they don't differ in the normal test as it is still `std::min(M, N) * batch_count`.